### ISSUE TITLE
Android build broken

### ISSUE THREAD

**Issue by whalefood** [Unity-Technologies/ml-agents#3167]
A project with mlagents won't compile when the platform is switched to android.  Getting a bunch of reference errors:

 "The type or namespace name 'Google' could not be found (are you missing a using directive or an assembly reference?)'

Looks like this is related to the google protobuf library.  You can see the issue just in the example project.

**Comment by surfnerd** [comment]
Hi @whalefood,
It looks like this DLL is included with Barracuda and is not available for those platforms according to their configuration in their package.  I have assigned @mantasp for him to take a look.

**Comment by surfnerd** [comment]
I believe this is related to #3125, which I have mistakenly closed.  I will reopen it.

**Comment by whalefood** [comment]
Thanks for taking a look

**Comment by surfnerd** [comment]
You may be able to copy it, and only include it for android.

**Comment by surfnerd** [comment]
So, I just did a sort of hacky thing and got it to work.  You can edit the Google.Protobuf.dll meta file.  Flip all of the Android platform enabled flags to 1.  Then you should be able to build for Android.

**Comment by surfnerd** [comment]
![Screen Shot 2020-01-06 at 2 52 45 PM](https://user-images.githubusercontent.com/1356616/71854820-34f53680-3094-11ea-8cbc-8c907e7ed9ad.png)

This is where I found that file.  You can open the `.meta` file with a text editor.

Modify the line
```
        Exclude Android: 0
```
to 
```
        Exclude Android: 1
``` 
and then modify 
```
  - first:
      Android: Android
    second:
      enabled: 0
      settings:
        CPU: ARMv7
```
to
```  - first:
      Android: Android
    second:
      enabled: 1
      settings:
        CPU: ARMv7
```

**Comment by surfnerd** [comment]
The same could be done for iOS

**Comment by whalefood** [comment]
Good to know.  Thanks!

**Comment by anttir** [comment]
![image](https://user-images.githubusercontent.com/481372/72206634-0cf14300-3499-11ea-9d6d-d46de879c338.png)
This fix worked for me, thank you! Just remember to Reimport the package after making the changes.

**Comment by akira675** [comment]
Is there a resolution for this issue that will work for Cloud Builds?

**Comment by JoRouss** [comment]
In unity 2019.3.3f1, ml-agent 0.14.1, I just had to update the Barracuda package to version 6.0.0 and I was able to build without the error.

**Comment by chriselion** [comment]
@JoRouss Good point, that was something that Barracuda addressed in the 0.6.0 release:
> Fix: enabled Google Protobuf for all platforms as ML Agents rely on this package. Fixes building ML Agents on non-desktop platforms.

(from https://github.com/Unity-Technologies/barracuda-release/blob/release/0.6.0/CHANGELOG.md#060---2020-02-14)
